Formula/bracken.rb

```diff
@@ -0,0 +1,28 @@
+class Bracken < Formula
+  # cite Lu_2017: "https://doi.org/10.7717/peerj-cs.104"
+  desc "Bayesian estimation species abundances from Kraken output"
+  homepage "https://ccb.jhu.edu/software/bracken/index.shtml"
+  url "https://github.com/jenniferlu717/Bracken/archive/v2.2.tar.gz"
+  sha256 "307495060d3575c3a90c87cf82074d4a63d20cf4062658f161951c60a66fd319"
+
+  depends_on "gcc" if OS.mac?
+
+  fails_with :clang # uses OpenMP
+
+  def install
+    cd "src" do
+      inreplace "Makefile", "g++", "$(CXX)"
+      system "make"
```

Comment:
If you delete the above line, you could instead use
```suggestion
      system "make", "-C", "src"
```

Reply:
Done.

Actually, I only just realised now that I get a install warning on macOS. I was primarily focusing on Linux.

```
 ❯ brew install bracken
==> Installing bracken from brewsci/bio
==> Downloading https://github.com/jenniferlu717/Bracken/archive/v2.2.tar.gz
Already downloaded: /Users/gabriel/Library/Caches/Homebrew/downloads/8582634cc4e28766810162dd38e7eb8a96469eb06b9e92332c5a7cddfd043f4f--Bracken-2.2.tar.gz
==> make
Warning: brewsci/bio/bracken dependency gmp was built with a different C++ standard
library (libc++ from clang). This may cause problems at runtime.
🍺  /usr/local/Cellar/bracken/2.2: 10 files, 155.5KB, built in 7 seconds
>>> elapsed time 14s
```

Any suggestion?

Reply:
You can safely ignore this warning.